src/docs/epics/44-migrate-to-maven-di/index.adoc

```diff
@@ -68,26 +70,46 @@ Getting rid of more Plexus dependencies is subject to other activities and beyon
 * [ ] Document <<sec:current-state,current state>>
 ** [ ] Check current documentation
 ** [ ] Evaluate dependencies, e.g., with https://jqassistant.org[jQAssistant]{nbsp}->{nbsp}This might also help for other Epics
+* [x] Situation JSR330/sisu vs maven-api-di/maven-di
 * [ ] Document new DI framework (https://github.com/apache/maven/tree/master/api/maven-api-di[Code] seems to have not even Javadocs)
 * [ ] Propose a migration path
 * [ ] Add automation where possible, e.g., via https://docs.openrewrite.org/[OpenRewrite]
 
 [[sec:current-state]]
 == Current state of DI migration
 
-TBD
+Maven 4 plugins should be migrated to maven-api-di.
```

Comment:
We should clarify what we mean by M3 and M4 plugins, e.g.,
* Plugins only working with either version
* Same plugin, different branches
* other (?)